code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm

```diff
@@ -235,8 +235,8 @@
 	radiation_mod = 0
 	toxins_mod = 3
 
-	grab_mod = 0.8
-	resist_mod = 4
+	grab_mod = 0.5 //very big, very easy to grab
+	resist_mod = 14 //also very strong
```

Comment:
This seems excessive, a G2 (which almost literally uses the force of an hydraulic clamp) is 12, and is far slower to move around than a Bulwark.

Reply:
The grab and resist mod is made this way due to size - Bulwarks and G2 are both much stronger than a human, but Bulwark size makes it easier for people to grab them and harder for them to keep a grip.